### PR TITLE
feat: Review organisations

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
Screenshot 2020-05-04 at 14 53 52

Key changes;
- Notifications for organisation in review in the menu widget
- Allow in review organisations to be approved/rejected
- Include a warning of what fields are duplicated from other orgs
- Split organisations into tabs based on status

### CODE REVIEW

File: ui_automation_tests/features/organisation.feature [hunk 1]
```diff
@@ -44,3 +44,37 @@ Feature: I want to add a company to LITE
...
+    When an organisation matching the existing organisation is created
+    And I go to the organisation
+    And I click review
+    Then I should be warned that this organisation matches an existing one
+
+  @LT_1105_review_and_reject_an_organisation @regression
+  Scenario: Review and approve an organisation
+    Given I sign in to SSO or am signed into SSO
+    And an in review organisation exists
+    When I go to organisations
```
Comment (phlptt) [inline ui_automation_tests/features/organisation.feature:73]: When should be And

File: ui_automation_tests/step_defs/test_organisation.py [hunk 2]
```diff
@@ -105,3 +108,105 @@ def click_organisation(driver, context):
 def click_edit(driver, context):
     OrganisationPage(driver).click_edit_organisation_link()
     OrganisationsFormPage(driver).fill_in_company_info_page_1(context)
+
+
+@given("an in review organisation exists")
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_organisation.py:113]: Can you change this to 
@given("I create an in review organisation")

File: ui_automation_tests/step_defs/test_organisation.py [hunk 3]
```diff
@@ -105,3 +108,105 @@ def click_organisation(driver, context):
...
+def in_review_organisation(context, api_test_client):
+    data = {
+        "name": "Org-" + get_current_date_time(),
+        "type": "commercial",
+        "eori_number": "1234567890AAA",
+        "sic_number": "12345",
+        "vat_number": "GB1234567",
+        "registration_number": "09876543",
+        "user": {"email": "name@example.com"},
+        "site": {"name": "site", "address": {"address_line_1": "Address-" + get_current_date_time()}},
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_organisation.py:123]: is there a reason this isn't in the submodule with the rest of the request data?

File: ui_automation_tests/step_defs/test_organisation.py [hunk 4]
```diff
@@ -105,3 +108,105 @@ def click_organisation(driver, context):
...
+
+@then("the organisation previously created is in the list")
+def organisation_in_list(driver, context):
+    assert find_paginated_item_by_id(context.organisation_id, driver)
+
+
+@when("I click review")
+def click_review(driver):
+    OrganisationPage(driver).click_review_organisation()
+    x = 1
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_organisation.py:154]: don't think this x = 1 is used anywhere

File: ui_automation_tests/features/organisation.feature (hunk 1 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/features/organisation.feature:73]: Does that not make it a given as opposed to a when?

File: ui_automation_tests/step_defs/test_organisation.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/step_defs/test_organisation.py:113]: Does that not imply that the internal user created the organisation when it's actually created by an anonymous user

File: ui_automation_tests/step_defs/test_organisation.py (hunk 4 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/step_defs/test_organisation.py:154]: lol forgot to remove that

File: ui_automation_tests/step_defs/test_organisation.py (hunk 2 above)
Comment (phlptt) [reply ui_automation_tests/step_defs/test_organisation.py:113]: ah i see. when i read @given("an in review organisation exists")
i just thought it was going to be something like
`if organisation doesn't exist
then create organisaion
else 
do nothing`

File: organisations/services.py [hunk 5]
```diff
@@ -38,6 +38,11 @@ def put_organisation(request, pk, json):
     return data.json(), data.status_code
 
 
+def put_organisation_status(request, pk, json):
+    data = put(request=request, appended_address=ORGANISATIONS_URL + str(pk) + "/status/", json=json)
```
Comment (stephanieboyd) [inline organisations/services.py:42]: status not a constant somewhere?

Comment (BenAndrew-Kainos) [reply organisations/services.py:42]: will add now

Comment (BenAndrew-Kainos) [reply organisations/services.py]: done

**Pull request merged**